<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QDialog" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableUsers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>129</y>
     <width>771</width>
     <height>321</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>12</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>129</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить список</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить пользователя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMakeAdmin">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Назначить администратора</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemoveAdmin">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Снять права администратора</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetPassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Сбросить пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExportCsv">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Экспорт CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlockUser">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Заблокировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUnblockUser">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Разблокировать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkaAdminsOnly">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Только администраторы</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBlockedOnly">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Только заблокированные</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
